front/src/modules/timesStops/types.ts

```diff
@@ -2,8 +2,16 @@ import type { TrainScheduleBase, TrainScheduleResult } from 'common/api/osrdEdit
 import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
 import type { ArrayElement } from 'utils/types';
 
-export type PathWaypointRow = SuggestedOP & {
+export type TimeExtraDays = {
+  time: string | null | undefined;
```

Comment:
Should it be a TimeString to know what we manipulate or it can be something else ?

```suggestion
  time?: TimeString | null;
```

Reply:
it’s only ever a TimeString, done